wiki/Beatmap/Beatmap_description/th.md

```diff
@@ -0,0 +1,21 @@
+# คำอธิบายบีทแมป
+
+**คำอธิบายบีทแมป** เป็นพื้นที่เล็ก ๆ ใต้หน้าของ [บีทแมพ](/wiki/Beatmap) ที่ให้ข้อมูลเกี่ยวกับบีทแมปแก่ผู้เล่นโดยใช้ [BBCode](/wiki/BBCode) โดยปกติแล้วจะเน้นข้อเท็จจริงบางอย่างเกี่ยวกับบีทแมปผู้สร้างที่ถือว่ามีความสำคัญเพียงพอ หรือใช้เพื่อโฆษณาโปรเจคบีทแมพอื่น ๆ
+
+## การใช้และข้อมูล
+
+คำอธิบายยอดนิยมของบีทแมพที่ถูก[จัดอันดับ](/wiki/Beatmap/Category#ranked) มักมีดังต่อไปนี้:
+
+- รายการ[ระดับความยากของแขก](/wiki/Beatmap/Guest_difficulty)และผู้สร้างทั้งหมด
+- ข้อมูลที่จะช่วยให้ผู้เล่นตัดสินใจว่าต้องการดาวน์โหลดบีทแมปหรือไม่
+- รายการแก้ไขหรือบันทึกการเปลี่ยนแปลง
+
+### รูปภาพ
+
+คำอธิบายบีทแมปจำนวนมากมีรูปภาพอย่างน้อยหนึ่งภาพ (มักเป็นภาพเคลื่อนไหว [GIF](https://en.wikipedia.org/wiki/GIF) หรืออะไรเช่นนี้) ซึ่งใช้เพื่อแสดงและอธิบายผู้สร้าง เพลง และ/หรือ บีทแมปอย่างฉูดฉาด พวกเขายังสามารถใช้เป็นชื่อหรือส่วนหัวเพื่อเพิ่มไหวพริบและคุณภาพ หากต้องการเรียนรู้วิธีเพิ่มรูปภาพ โปรดดูที่ [บทความของ BBCode หัวข้อเกี่ยวกับรูปภาพ](/wiki/BBCode#image)
```

Comment:
```suggestion
คำอธิบายบีทแมพจำนวนมากมีรูปภาพอย่างน้อยหนึ่งภาพ (มักเป็นภาพเคลื่อนไหว [GIF](https://en.wikipedia.org/wiki/GIF) หรืออะไรเช่นนี้) ซึ่งใช้เพื่อแสดงและอธิบายผู้สร้าง เพลง และ/หรือ บีทแมพอย่างชัดเจน พวกเขายังสามารถใช้เป็นชื่อหรือส่วนหัวเพื่อเพิ่มความสวยงามและคุณภาพ หากต้องการเรียนรู้วิธีเพิ่มรูปภาพ โปรดดูที่ [บทความของ BBCode หัวข้อเกี่ยวกับรูปภาพ](/wiki/BBCode#image)
```
flair in this context probably means stylishness and originality. not instinct thing